CHANGES.rst

```diff
@@ -11,6 +11,9 @@ Release Notes
 
 - Set Python min version to 3.8. [#219]
 
+- Fix problem problem reported where angle of 180 degrees results in an
```

Comment:
```suggestion
- Fix problem reported where angle of 180 degrees results in an
```

Reply:
I think "problem problem" means a difficult problem

Reply:
like "problematic problem"

Reply:
Doesn't the first "problem" negate the second "problem," like a double negative? 😉

Reply:
No, I think it enhances it. "problem problem" == "hard problem"